ansible/roles/ansible_service_broker_setup/tasks/main.yml

```diff
@@ -78,7 +78,7 @@
       url: http://{{ ansible_service_broker_route }}/v2/bootstrap
       method: POST
     register: response
-    until: response|success and response.json and response.json.spec_count
+    until: response|success and response.json and response.json.spec_count is defined
```

Comment:
This change keeps catasb from failing whenever someone doesn't have any apbs yet.